owl/SOMA-HOME.owl

```diff
@@ -12,59 +12,43 @@ Ontology(<http://www.ease-crc.org/ont/SOMA-HOME.owl>
 Import(<http://www.ease-crc.org/ont/SOMA.owl>)
 Annotation(rdfs:comment "SOMA-HOME defines the concepts that are specific to home, e.g. kitchen environment, household items.")
 
-Declaration(Class(:Box))
-Declaration(Class(:BreadKnife))
-Declaration(Class(:BreakfastPlate))
-Declaration(Class(:Carafe))
-Declaration(Class(:CerealBox))
-Declaration(Class(:ClosingDisposition))
-Declaration(Class(:CoffeeCarafe))
-Declaration(Class(:Cup))
-Declaration(Class(:DessertFork))
-Declaration(Class(:DinnerPlate))
-Declaration(Class(:Dispenser))
-Declaration(Class(:Glass))
-Declaration(Class(:JamJar))
-Declaration(Class(:Jar))
-Declaration(Class(:Kitchen))
-Declaration(Class(:KitchenKnife))
-Declaration(Class(:MilkBottle))
-Declaration(Class(:MilkPack))
-Declaration(Class(:NutellaJar))
-Declaration(Class(:Oven))
-Declaration(Class(:PastaBowl))
-Declaration(Class(:PepperShaker))
-Declaration(Class(:RaspberryJamJar))
-Declaration(Class(:SaladBowl))
-Declaration(Class(:SaltShaker))
-Declaration(Class(:Shaker))
-Declaration(Class(:SoupPot))
-Declaration(Class(:Spoon))
-Declaration(Class(:SugarDispenser))
-Declaration(Class(:TableFork))
-Declaration(Class(:TableKnife))
-Declaration(Class(:TableSpoon))
-Declaration(Class(:TeaSpoon))
-Declaration(Class(:TemperingByCooling))
-Declaration(Class(:TemperingByHeating))
-Declaration(Class(:TrashContainer))
-Declaration(Class(:WaterBottle))
-Declaration(Class(:WaterGlass))
-Declaration(Class(:WineBottle))
-Declaration(Class(:WineGlass))
+Declaration(Class(<http://www.ease-crc.org/ont/SOMA-HOME.owl#Armchair>))
```

Comment:
I'm a bit confused by the prefixes situation.

New concepts appear here with SOMA-HOME.owl in the prefix. At the same time, something like NutellaJar (seems too specific to be in SOMA proper) uses SOMA.owl as prefix. This is not an issue with this PR as such, rather it is something to keep in mind, perhaps by opening a github issue.

Reply:
There is a related issue here: https://github.com/ease-crc/soma/issues/288